pkg/kapp/clusterapply/waiting_changes.go

```diff
@@ -119,7 +120,11 @@ func (c *WaitingChanges) WaitForAny() ([]WaitingChange, error) {
 		}
 
 		if time.Now().Sub(startTime) > c.opts.Timeout {
-			return nil, fmt.Errorf("Timed out waiting after %s", c.opts.Timeout)
+			var timedOutResources []string
+			for _, change := range c.trackedChanges {
+				timedOutResources = append(timedOutResources, change.Graph.Change.Resource().Description())
```

Comment:
change.ClusterChange.Resource().Description() is a bit shorter and probably better in general because we are working with cluster changes.

Reply:
Updated